CMakeLists.txt

```diff
@@ -1,10 +1,275 @@
 # Native code side of Thrive
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.13)
+
+set(CMAKE_CONFIGURATION_TYPES "Debug;Release;RelWithDebInfo;Distribution" CACHE STRING "" FORCE)
```

Comment:
You should copy the comment from where you moved this code (the reason why the non-standard distribution type is here).

Reply:
done, i added something to it too :]

Reply:
I still don't see the comment text about Jolt requiring the Distribution type...